code/game/gamemodes/objective.dm

```diff
@@ -427,6 +431,22 @@ GLOBAL_LIST_INIT(potential_theft_objectives, (subtypesof(/datum/theft_objective)
 		if(I.type in steal_target.altitems)
 			return steal_target.check_special_completion(I)
 
+/datum/objective/steal/proc/give_supermatter_kit()
+	var/mob/living/carbon/human/mob = owner.current
+	var/T = new /obj/item/storage/box/syndie_kit/supermatter
```

Comment:
`T` is typically used to refer to turfs, so I'd probably rename this to `I` or something similar

Reply:
Changed